cf-deployment.yml

```diff
@@ -66,12 +66,12 @@ instance_groups:
           domain: cf.internal
           servers: &3
             lan: *1
-        encrypt_keys: "((consul_encrypt_keys))"
-        agent_cert: "((consul_agent_cert))"
-        agent_key: "((consul_agent_agent_key))"
-        ca_cert: "((consul_agent_ca_cert))"
-        server_cert: "((consul_agent_server_cert))"
-        server_key: "((consul_agent_server_key))"
+        encrypt_keys: [((consul_encrypt_key))]
```

Comment:
The above is an array consisting of a single value.  The [`cf-filler` recipe](https://github.com/cloudfoundry/cf-deployment/blob/master/cf-filler/recipe-cf-deployment.yml) similarly only provides a password array with a single value.  It seems like right now, either you don't leverage something for automatic credential generation so that you can easily manage credential rotation yourself in your `-l` file, making the non-rotation and rotation deploys have similar UX, or you leverage automatic generation making the basic deploy easy but have to think about it a fair bit more when trying to do rotation.

Long run, it might be better for Config Server to support a password list type (instead of just a password type), but for this to be useful a lot of upstream code will need to change to respect password rotation via multiple passwords.